### ISSUE TITLE
Change wok/kimchi port to 443

### ISSUE THREAD

**Issue by kosli** [kimchi-project/kimchi#1154]
Hi 

I changed the auto-generated nginx wok.conf (ubuntu with latest wok & kimchi packages) to use https on standard port 443. access to the wok web interface itself works fine via https://server/, but as soon as i click on "view console" of a VM, it sends me to https://server:8001/plugins/kimchi/spice_auto.html?port=8001  
Where do I need to change the port in kimchi? shouldn't it auto-generate the URL with the correct settings?

Thanks
 KoS

**Comment by alinefm** [comment]
Hi @kosli 

You need to update the port on /etc/wok/wok.conf file as well.

**Comment by kosli** [comment]
@alinefm 
are you sure? because as soon as un-comment the setting proxy_port = 8001 (and change it to 443), wok itself will try to listen on that socket, but i am using nginx (and would like to manually configure it to my needs) ?

**Comment by alinefm** [comment]
Yeap! As you can notice, Wok needs to know the proxy port, ie, the port on which nginx is running.
So once you have already updated nginx config to listen on 443, you need to update wok.conf file as well.

**Comment by kosli** [comment]
thanks, solved the problem. i was just puzzled that when i originally changed the setting in wok.conf to 443 and tried to restart wokd service i got an error message that the port is already in use.

now i have another weird problem which wasn't there before. in the preview windows the VM looks okay, but when I use "view console" the screen is mirror vertically and horizontally. I don't think this has anything to with the changed port?! 

looks quite funny ;-)
![image](https://user-images.githubusercontent.com/203928/27838169-31dda5b8-60e9-11e7-95dd-ca35679e751d.png)

**Comment by alinefm** [comment]
Is it a Win guest? I think there is an open issue related to that.

**Comment by kosli** [comment]
yes, it is a windows guest. haven't seen that issue on other KVM servers that i have running, but would need to check. so you say it could be a problem of the 2.5.0 kimchi release?

**Comment by alinefm** [comment]
@kosli It is a problem on graphic tool. Have you tried VNC instaed Spice?

**Comment by kosli** [comment]
@alinefm switch to VNC in virt-manager makes it worse:

- no screenshot in kimchi as preview
- instead of the console i get a "Server disconnected (code: 1006)"

**Comment by luciantugui** [comment]
same for me on 443, "Server disconnected (code: 1006)"

**Comment by YoSmartInc** [comment]
> same for me on 443, "Server disconnected (code: 1006)"



> @alinefm switch to VNC in virt-manager makes it worse:
> 
> * no screenshot in kimchi as preview
> * instead of the console i get a "Server disconnected (code: 1006)"

Meet this issue too, had this been fixed?
